tasks/setup-Debian.yml

```diff
@@ -32,7 +32,9 @@
   shell: "update-java-alternatives -l | grep openjdk | cut -f -1 -d ' '"
   changed_when: false
   register: openjdk_alternative
+  when: not ansible_check_mode
```

Comment:
`shell` and `command` doesn't handle `check_mode`: aren't these tasks already skipped when check mode is enabled.

Reply:
I would rather use `check_mode: no` with the first task.

Reply:
OK. I have fixed this. :+1: